### PR TITLE
Fix URLs (remove reference to lovelace)

### PR DESCRIPTION
**Pull request by cnotin**
These seem to be the new correct URLs

### CODE REVIEW

File: docs/notifications/actionable.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ action:
 
 ### `uri` values
 
-To navigate to a frontend page, use the format `/lovelace/test` where `test` is replaced by your defined [`path`](https://www.home-assistant.io/lovelace/views/#path) in the defined view. If you plan to use a lovelace dashboard the format would be `/lovelace-dashboard/view` where `/lovelace-dashboard/` is replaced by your defined [`dashboard`](https://www.home-assistant.io/lovelace/dashboards/) URL and `view` is replaced by the defined [`path`](https://www.home-assistant.io/lovelace/views/#path) within that dashboard. For example:
+To navigate to a frontend page, use the format `/lovelace/test` where `test` is replaced by your defined [`path`]([https://www.home-assistant.io/dashboards/views/#path](https://www.home-assistant.io/dashboards/views/#path)) in the defined view. If you plan to use a lovelace dashboard the format would be `/lovelace-dashboard/view` where `/lovelace-dashboard/` is replaced by your defined [`dashboard`](https://www.home-assistant.io/dashboards/dashboards) URL and `view` is replaced by the defined [`path`](https://www.home-assistant.io/dashboards/views/#path) within that dashboard. For example:
```
Comment (frenck) [inline docs/notifications/actionable.md:104]:
```suggestion
To navigate to a frontend page, use the format `/lovelace/test` where `test` is replaced by your defined [`path`](https://www.home-assistant.io/dashboards/views/#path) in the defined view. If you plan to use a dashboard the format would be `/lovelace-dashboard/view` where `/lovelace-dashboard/` is replaced by your defined [`dashboard`](https://www.home-assistant.io/dashboards/dashboards) URL and `view` is replaced by the defined [`path`](https://www.home-assistant.io/dashboards/views/#path) within that dashboard. For example:
```

**Comment by cnotin** [comment]
Oops sorry for the doubled markdown!

**Comment by frenck** [comment]
No worries! That is something that happens magically now when copy & pasting on GitHub (which is quite annoying at times to be honest).

../Frenck

**Review by frenck — APPROVED** [review]
Thanks, @cnotin 👍 

../Frenck

**Pull request merged**